packages/shared/friends/sagas.ts

```diff
@@ -239,25 +240,60 @@ function* configureMatrixClient(action: SetMatrixClient) {
   yield Promise.all(
```

Comment:
Maybe wrap this in a try catch to prevent bubbling up the exception to a fatal error
```suggestion
  yield Promise.all(
```